### PR TITLE
additional styling

### PR DESCRIPTION
**Pull request by RichVelo**
I have accidently not set up another task branch

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -14,7 +14,9 @@
 echo '<body>';
 
 
-echo '<h1>Richie\'s whips</h1>';
+echo '<div class="pageTitle">' . '<h1>Richie\'s whips</h1>' . '</div>';
```
Comment (maxwell-01) [inline index.php:17]: you should create an html page and only echo the functions you need

File: style.css [hunk 2]
```diff
@@ -1,21 +1,41 @@
+html {
+    font-size: 10px;
+}
+
+.pageTitle {
```
Comment (maxwell-01) [inline style.css:5]: this should be a header section not a div

File: style.css [hunk 3]
```diff
@@ -1,21 +1,41 @@
...
+    display: flex;
+    justify-content: center;
+}
+h1 {
+    font-size: 3rem;
+    justify-content: center;
+}
+
 .bikeCards {
```
Comment (maxwell-01) [inline style.css:14]: this should be in main and be a section not div

File: style.css [hunk 4]
```diff
@@ -1,21 +1,41 @@
...
     border: 2px solid #000000;
     width: 15%;
 }
 
 .container {
-    width: 100vw;
+    margin: 0 auto;
+    width: 85vw;
```
Comment (maxwell-01) [inline style.css:21]: its better practice to specify the max width rather than use vw in this way

File: style.css [hunk 5]
```diff
@@ -1,21 +1,41 @@
...
+    flex-direction: row;
```
Comment (maxwell-01) [inline style.css:22]: row is the default, you dont need to write it in

File: style.css [hunk 6]
```diff
@@ -1,21 +1,41 @@
...
+    width: 80%
 }
 
-/*.bike {*/
-/*    padding: 10px;*/
-/*    margin: 10px;*/
-/*}*/
\ No newline at end of file
+
+@media screen and (max-width: 650px) {
```
Comment (maxwell-01) [inline style.css:33]: these are not working as intended 
Screenshot 2022-02-08 at 12 31 00

File: index.php (hunk 1 above)
Comment (RichVelo) [reply index.php:17]: I'm assuming this is by closing off the php then using the html as normal - I've now done this :)

File: style.css (hunk 3 above)
Comment (RichVelo) [reply style.css:14]: changed

File: style.css (hunk 2 above)
Comment (RichVelo) [reply style.css:5]: changed

File: style.css (hunk 5 above)
Comment (RichVelo) [reply style.css:22]: changed

File: style.css (hunk 6 above)
Comment (RichVelo) [reply style.css:33]: it does at 650px - but I take your point that theres overflow and readability isn't great

Comment (RichVelo) [reply style.css:33]: I have a solution :D

Comment (RichVelo) [reply style.css:33]: 🗡️

**Review by maxwell-01 — COMMENTED** [review]
I'd recommend going back and looking through the notes (or another site you made) to check the html structure you need for the page

File: index.php [hunk 7]
```diff
@@ -10,17 +10,18 @@
...
+?>
 
-echo '<div class="pageTitle">' . '<h1>Richie\'s whips</h1>' . '</div>';
+<html>
 
 
-echo '<div class="container">';
-echo bikeDetails($allBikes);
-echo '</div>';
+<header class="pageTitle"><h1>Richie's whips</h1></header>
```
[the remark this answers is not in the document]
Comment (RichVelo) [reply index.php:19]: this was to allow for future change but removed :)

File: index.php [hunk 8]
```diff
@@ -10,17 +10,18 @@
 $query->execute();
 
 $allBikes = $query->fetchAll();
-echo '<html>';
-echo '<body>';
 
+?>
 
-echo '<div class="pageTitle">' . '<h1>Richie\'s whips</h1>' . '</div>';
+<html>
```
[the remark this answers is not in the document]
Comment (RichVelo) [reply index.php:16]: added

File: index.php [hunk 9]
```diff
@@ -10,17 +10,18 @@
...
```
[the remark this answers is not in the document]
Comment (RichVelo) [reply index.php:17]: done

**Review by maxwell-01 — APPROVED** [review]

**Pull request merged**